src/prefect/tasks/prefect/flow_run_cancel.py

```diff
@@ -0,0 +1,38 @@
+from prefect import Task
+from prefect.client import Client
+from typing import Any
+import prefect
+
+
+class CancelFlowRunTask(Task):
+    """
+    Task to cancel a flow.
+    Args:
+        - flow_run_id_to_cancel (str, optional): The ID of the flow to cancel
```

Comment:
Let's change this to just `flow_run_id` since the Cancel is part of the task name